ReportingServicesTools/Functions/CatalogItems/New-RsSubscription.ps1

```diff
@@ -313,7 +371,9 @@ function New-RsSubscription
         }
         catch
         {
-            throw (New-Object System.Exception("Exception occurred while creating subscription! $($_.Exception.Message)", $_.Exception))
+            $ex = $_.Exception;
+            $ErrorMessage = $ex.Message;
+            throw (New-Object System.Exception("Exception occurred while creating subscription! ErrorMessage: [$ErrorMessage]", $Exception))
```

Comment:
not mandatory , but it would be great if tests for this new functionality are added, the coverage there is already pretty low

Reply:
I might be able to add tests next week but it already took me a long time to fix everything.

Can you update the README.md with instructions on how to set-up a brand new PC with SSRS using the ReportingServicesTools?  As an outsider, it was very hard even figuring out the right scaffolding steps to deploy Reports, much less configure Report Server. There is strange vocabulary in the ReportingServicesTools Admin synposes, like references to "PBIRS," which I don't use and that intimidates me.

Reply:
The most up to date sample on how to configure it using the powershell tools is referencing the continuous integration script , where it uses the powershell tools to configure the test machine https://github.com/Microsoft/ReportingServicesTools/blob/d06abfc41ae76c51eb548574972498760afe9605/appveyor.yml#L66 

The original idea of the admin cmdlets were created mostly to enable automation, feel free to add more samples or propose changes to make it more usable for all the users.

And for PBIRS it stands for Power BI Report Server which is a superset of Reporting Services (https://docs.microsoft.com/en-us/power-bi/report-server/get-started)

Reply:
@jtarquino  I think the part that confused me is in a lot of the Admin powershell functions it refers to PBIRS instead of just the subset of Reporting Services I might colloquially call "SSRS". What do you think about clarifying that part of the documentation?

Reply:
So, are these enhancements already applied to the current version of ReportingServicesTools?
If they are not: what are we waiting for? I really need them and already tested\used in my environment, but "clumsy" way.